test/13_loki-backend.ts

```diff
@@ -99,6 +99,7 @@ describe('loki backend', () => {
 		const loki = new LokiBackend();
 		const log = createLog();
 		const incomingLog = await new Bluebird(async (resolve) => {
+			// eslint-disable-next-line @typescript-eslint/no-floating-promises
 			loki.subscribe(ctx, resolve);
```

Comment:
Why woudln't await make sense here? Would it even make the `await setTimeout(100)` obsolete?

Reply:
I actually had this await originally but @Page- was concerned about it affecting the timings, so I decided to keep this a no-op.
See: https://github.com/balena-io/open-balena-api/pull/1396#discussion_r1301675512